singer_sdk/testing/target_tests.py

```diff
@@ -108,6 +110,11 @@ class TargetRecordMissingKeyProperty(TargetFileTestTemplate):
 
     name = "record_missing_key_property"
 
+    def test(self) -> None:
+        """Run test."""
+        with pytest.raises(MissingKeyPropertiesError):
+            super().test()
```

Comment:
I'm a bit confused by this. Does it mean that the test succeeds if the record is missing a key property?

Reply:
@edgarrmondragon this tests that the SDK implementation correctly raises a `MissingKeyPropertiesError` if the Tap sends a `RECORD` with missing key properties 🙂 Previously we were throwing a generic `Exception`, so this just makes the error more user-friendly.